### ISSUE TITLE
Problem with @nuxt/font Usage

### ISSUE THREAD

**Issue by bukandicki** [nuxt/fonts#270]
**Environment:**
- **Node Version:** v18.20.3
- **Nuxt Version:** ^3.13.0
- **@nuxt/fonts Version:** ^0.8.0
- **Package Manager:** pnpm 9.6.0

**Description:**
I am experiencing an issue with the font definitions generated by the `@nuxt/font` package in my Nuxt 3 application. The package generates CSS that improperly uses the `local()` function in the `src` property of the `@font-face` declarations, causing the fonts to not load correctly.

**Generated CSS Snippet:**
```css
@font-face {
    font-display: swap;
    font-family: 'Archivo';
    font-weight: 300;
    src: local("url('/fonts/Archivo-Light.woff2')");
}

/* Other generated font definitions... */
```

**Expected Behavior:**
The fonts should load correctly without any issues.

**Actual Behavior:**
The generated CSS prevents the fonts from loading properly due to the incorrect syntax in the `src` property.

**Nuxt Configuration:**
```ts
fonts: {
    defaults: {
      styles: ["normal"],
    },
    families: [
      {
        name: "Archivo",
        src: "url('/fonts/Archivo-Light.woff2')",
        global: true,
        weight: 300
      },
      {
        name: "Archivo",
        src: "url('/fonts/Archivo-Regular.woff2')",
        global: true,
        weight: 400
      },
      {
        name: "Archivo",
        src: "url('/fonts/Archivo-Medium.woff2')",
        global: true,
        weight: 500
      },
      {
        name: "Archivo",
        src: "url('/fonts/Archivo-SemiBold.woff2')",
        global: true,
        weight: 600
      },
      {
        name: "Clash Display",
        src: "url('/fonts/ClashDisplay-Regular.woff2')",
        global: true,
        weight: 400
      },
      {
        name: "Clash Display",
        /*
          Even when i following the docs with this format (no quote)
          src: { name: 'Other Font', src: 'url(/fonts/ClashDisplay-Medium.woff2)', weight: 'bold' }
        */
        src: "url('/fonts/ClashDisplay-Medium.woff2')",
        global: true,
        weight: 500
      },
    ]
  }
```

**Steps to Reproduce:**
1. Install `@nuxt/font` using pnpm.
2. Configure the module in `nuxt.config.js` with the provided font settings.
3. Run the application.

**Additional Context:**
- The paths to the font files are correct, but the current usage of `local()` in the generated CSS is causing loading issues.
- A potential fix could be to modify the generated CSS to remove `local()` and use `url()` directly:
  ```css
  src: url('/fonts/Archivo-Light.woff2') format('woff2');
  ```

Thank you for your assistance!
@danielroe

**Comment by danielroe** [comment]
You should not set `src: "url('/fonts/Archivo-Light.woff2')"`.

Instead, you set `src: "/fonts/Archivo-Light.woff2"`.

Fixed the incorrect docs here: https://github.com/nuxt/fonts/commit/ca44e0d274518f1ab37ddba20945f9354dd669da.

Let me know if you encounter any further issues 🙏

**Comment by bukandicki** [comment]
> You should not set `src: "url('/fonts/Archivo-Light.woff2')"`.
> 
> Instead, you set `src: "/fonts/Archivo-Light.woff2"`.
> 
> Fixed the incorrect docs here: [ca44e0d](https://github.com/nuxt/fonts/commit/ca44e0d274518f1ab37ddba20945f9354dd669da).
> 
> Let me know if you encounter any further issues 🙏

Thank you for the quick response! I updated the src property to src: "/fonts/Archivo-Light.woff2" as you suggested, and it resolved the issue. The fonts are now loading correctly.

I appreciate the fix in the documentation as well. If I encounter any further issues, I'll be sure to reach out. Thanks again! 🙏
